packages/apps/patterns/react-ui/src/translations/locales/en-US.ts

```diff
@@ -2,4 +2,7 @@
 // Copyright 2022 DXOS.org
 //
 
-export const os = {};
+export const os = {
```

Comment:
I understand why you named this `os` but I'm conflicted because it breaks the pattern of it following the package name

Reply:
`ui` just seemed… relatively more-incorrect to me, I suppose? I'm amenable to any better name we might prefer.

Reply:
Or maybe if we prefix all of our namespaces with `dxos` it'd be fine, which is why the package names are the same way. Maybe we could do `dxos-ui`, `dxos-appkit`, etc.